src/components/RightHandMenu/RightHandMenu.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+
+import './RightHandMenu.css'
+import ToggleSelect from './ToggleSelect/ToggleSelect'
+import RadioSelect from './RadioSelect/RadioSelect'
+import Donut from './DonutChart/Donut'
+import UnequalDonut from './DonutChart/UnEqualDonut/UnequalDonut'
+
+// Static Content to show in right hand menu
+const dataTypes = {
+  'Poverty Rates' : {
+    'desc': 'Text about poverty rates and the data and possibly the next year',
+    'toggleSelect': ['Total', 'Children'],
+    'radioSelect': null
+  },
+  'Food Insecurity' : {
+    'desc': 'Text about food insecurity rates and the data and possibly the next year',
+    'toggleSelect': ['Total', 'Children'],
+    'radioSelect': {
+      'Total' : ['2018', '2020 (Projected)'],
+      'Children' : ['2018', '2020 (Projected)'] 
+      }
+  },
+  'WIC Usage' : {
+    'desc': 'Text about WIC usage data and the data and possibly the next year',
+    'toggleSelect': ['Enrollment', 'Race'],
+    'radioSelect': {
+      'Enrollment' : null,
+      'Race' : ['Women', 'Infants', 'Children'],
+      }
+  },
+  'Census Data' : {
+    'desc': 'Text about Census data and the data and possibly the next year',
+    'toggleSelect': null,
+    'radioSelect': null
+  }
+}
+
+/*
+ * COMPONENT: RightHandMenu
+ */
+
```

Comment:
For your reference, I think what we will do when getting the data will be to have a selectedCounty field in the store (similar to hoveredCounty right now) - and that will be the 6-digit ID. We can use that ID to use a selector to get the data we need from that county from the store. And we'll probably set the data sets we want as strings based on input from the left-bottom component so we can use that to select which data grouping we want as well.

This can all be adjusted in a later PR, because I think what you have is on the right track.

Reply:
☝️☝️☝️ Already working on it.... Next PR